src/cell.h

```diff
@@ -237,12 +237,12 @@ struct Cell {
                 str.Prepend(wxString() << text.stylebits);
                 str.Prepend(L" stylebits=\"");

```

Comment:
please also check for these `stylebits`

Reply:
Ok please see last commit. It XORs the stylebits from child and parent. Only if they differ, stylebits will be emitted in XML export.